components/formats-bsd/src/loci/formats/AxisGuesser.java

```diff
@@ -71,18 +73,24 @@
   public static final int S_AXIS = 4;
 
   /** Prefixes indicating space dimension. */
-  public static final String[] Z_PREFIXES = {
+  public static final ImmutableSet<String> Z_PREFIXES = ImmutableSet.of(
```

Comment:
I'm very hesitant to use external libraries unless it's necessary, and not just for the public API. In this case, however, I couldn't find a proper immutable collection in the standard Java API. There's [Collections.unmodifiableSet](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#unmodifiableSet(java.util.Set)), but it's just an unmodifiable view of an underlying collection that's still mutable. Would that be enough? Or maybe there is another solution I'm not aware of. Any suggestions?